### PR TITLE
Create new orb for e2e tests

### PR DESCRIPTION
**Pull request by qvalero**
I have created new orb for e2e tests execution on dev repo.
I have tested it on ui-jobteaser repo and execution is doing correctly: 
https://app.circleci.com/pipelines/github/jobteaser/ui-jobteaser/1937/workflows/e8465e6d-055c-4ece-a2cb-393f2c326075

I will do the PR on other repo once this one is merged and master is tag

### CODE REVIEW

File: orbs/e2e-web/orb.yml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        default: $COOKIE_E2E
+    steps:
+      - run:
+          working_directory: e2e_jt_tests
+          name: "run the test suite with chrome on staging"
+#https://circleci.com/docs/2.0/configuration-reference/#default-shell-options
+# in case of failure, do not mark step as failed (experiment of e2e tests on this repo)
+          command: |
+            export BYPASS_RACK_ATTACK=<< parameters.e2e_cookie >>
+            << parameters.command >>
```
Comment (happypoulp) [inline orbs/e2e-web/orb.yml:83]: **remark**

I am wondering if you could not hardcode `npm run chrome` here and only parameterize the eventual env vars that goes before it.

Comment (happypoulp) [reply orbs/e2e-web/orb.yml:83]: So `parameters.command` would become `parameters.command_envs` for ex.

**Review by happypoulp — APPROVED** [review]
👍  LGTM, well done for your first Orb 💪 ! (just one remark)

File: orbs/e2e-web/orb.yml (hunk 1 above)
Comment (qvalero) [reply orbs/e2e-web/orb.yml:83]: We have several command like `npm run firefox` `npm run chrone:master` `npm run chrome:de`... I think it is better to let the command free to parameter in dev project depending on the need.

**Pull request merged**